integration_tests/publish_intent_requests_test.go

```diff
@@ -110,6 +110,21 @@ var _ = Describe("Publish Intent Requests", func() {
 					expectedResponse = HTTPTestResponse{Code: 409, Body: urlArbiterResponseBody}
 					assertSameResponse(actualResponse, &expectedResponse)
 				})
+
+				It("returns a 500 error with a custom response", func() {
+					urlArbiterResponseCode = 418
+					urlArbiterResponseBody = `{ "message": "I'm a teapot"}`
+
+					actualResponse := doRequest("PUT", endpoint, publishIntentPayload)
+
+					Expect(testURLArbiter.ReceivedRequests()).To(HaveLen(1))
+					Expect(testDraftContentStore.ReceivedRequests()).To(BeEmpty())
+					Expect(testLiveContentStore.ReceivedRequests()).To(BeEmpty())
+
+					expectedResponseBody := `{"message": "Unexpected error whilst registering with url-arbiter"}`
+					expectedResponse = HTTPTestResponse{Code: 500, Body: expectedResponseBody}
+					assertSameResponse(actualResponse, &expectedResponse)
+				})
```

Comment:
this test makes me wonder whether there's a case for extracting out a `url_arbiter_error_response_test.go`, where publishing-api behaviour in response to url arbiter response is tested. i don't think we expect this behaviour to change irrespective of which endpoint the API consumer contacts.

maybe, i shouldn't have replicated these tests (url arbiter error response tests) across endpoints to begin with. i'm happy if we have to take care of this as a separate PR, just thought this was a nice example to discuss this change.

what do you think @jamiecobbett?